thunder/__init__.py

```diff
@@ -779,10 +779,10 @@ def wrapped(*args, **kwargs):
         return wrapped
 
     get_computation_and_inputs = decorate_computation_function(get_computation_and_inputs, host_execution_timer)
-    cd.get_computation_and_inputs = get_computation_and_inputs
-    cd.populate_cache_info = populate_cache_info
-    cd.acquire_initial_trace = acquire_initial_trace
-    cd.apply_transforms_and_build_cache_entry = apply_transforms_and_build_cache_entry
+    # cd.get_computation_and_inputs = get_computation_and_inputs
+    # cd.populate_cache_info = populate_cache_info
+    # cd.acquire_initial_trace = acquire_initial_trace
+    # cd.apply_transforms_and_build_cache_entry = apply_transforms_and_build_cache_entry
```

Comment:
Some tests rely on these and they seem publicly available.

Reply:
Ugh. Thorny. We could make cd an object with these methods if it helps. Or we could replace the capture of cd with a weakref.